copyright

```diff
@@ -1,4 +1,4 @@
-Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+﻿mat: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
```

Comment:
smh

Reply:
If this happen again I swear I'm gonna commit a forkicide